util/date/util.go

```diff
@@ -0,0 +1,33 @@
+package date
```

Comment:
maybe rename package to `dateutil`?

Reply:
and add simple tests

Reply:
I will add tests on https://linear.app/anytype/issue/GO-4451/support-seconds-precision-date-object-id as date Id format will be changed